core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -280,6 +289,19 @@ sealed abstract private[data] class OneAndLowPriority0_5 extends OneAndLowPriori
 }
 
 sealed abstract private[data] class OneAndLowPriority0 extends OneAndLowPriority0_5 {
+  implicit def catsDataPartialOrderForOneAnd[A, F[_]](implicit
```

Comment:
You can put this together with `Order` because it's a supertype.
So if `F[x] <: G[x]` it's ok to put them on the same level.
But if you were to add `Hash` then it has to be on a separate level from `Order` because `Eq` is common between `Hash` and `Order` but neither is a subtype of the other.

Reply:
Thank you for the clarification 👍  Personally, I would like to gather them in one place and I was considering to.
However, I found out that looks like almost everywhere in Cats `Order` and `PartialOrder` are placed on different levels. Perhaps just due to an oversight though (many oversights, actually). So I was uncertain about it and decided to follow the convention – I thought it was made for a reason. But if there's no objections, I would put it together indeed.

Reply:
Since those are package private traits we are free to reorder them 👍

Reply:
Ahh damn, I think it's a bug when there are higher-kinded type parameters. Got it, thanks 👍